<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generar Reporte</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/btn11.ico</normaloff>../resources/btn11.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reporte</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video2. TRAYECTORIA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoIpn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/logoipn.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoEscom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/logoescom.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPDF">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>570</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/btn12.png</normaloff>../resources/btn12.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcess">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>560</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblText_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre de la Prueba:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duración de la Prueba:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descripción de la Prueba:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombrePrueba">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDuracionPrueba">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtDescripcionPrueba">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>630</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
